cranelift/wasm/src/code_translator.rs

```diff
@@ -786,47 +789,47 @@ pub fn translate_operator<FE: FuncEnvironment + ?Sized>(
             );
         }
         Operator::V128Load8x8S { memarg } => {
-            let (flags, base) = unwrap_or_return_unreachable_state!(
+            let (flags, _, base) = unwrap_or_return_unreachable_state!(
```

Comment:
Let's add a TODO referring to a filed issue to add the load/store hooks for vector operands -- something like `// TODO(#1234): add `before_load` / `before_store` for SIMD loads and stores.` where `1234` is a GitHub issue number for a new issue you've filed.